.github/workflows/clean-up-after-branch-merge.yaml

```diff
@@ -0,0 +1,46 @@
+on:
+    pull_request:
+        types:
+            - closed
+name: Clean up after branch merge
+jobs:
+    variables:
+        runs-on: ubuntu-22.04
+        outputs:
+            BRANCH_NAME: ${{ steps.variables.outputs.BRANCH_NAME }}
+        steps:
+            -   name: Extract variables to output
+                id: variables
+                run: |
+                    BRANCH_NAME=${GITHUB_HEAD_REF:-${GITHUB_REF#refs/heads/}}
+                    echo "BRANCH_NAME=${BRANCH_NAME}" >> $GITHUB_OUTPUT
+    cancel-review-after-branch-merge:
+        name: Cancel review after branch merge
+        runs-on: [self-hosted, linux, review-stage]
+        needs: variables
+        steps:
+            -   name: GIT checkout branch - ${{ vars.MAIN_BRANCH_NAME }}
+                uses: actions/checkout@v3
```

Comment:
It would be nice to unite version of checkout, ideally through all workflows

Reply:
now we use version 4